indico/modules/categories/serialize.py

```diff
@@ -18,7 +18,10 @@
 from indico.util.string import sanitize_html
 
 
-def serialize_categories_ical(category_ids, user, event_filter=True, event_filter_fn=None, update_query=None):
+def serialize_categories_ical(
+    category_ids, user, event_filter=True, event_filter_fn=None,
+    update_query=None, reminder_minutes=None
```

Comment:
```suggestion
    update_query=None, *, reminder_minutes=None
```

For arguments like this one it's a good idea to make them keyword-only.